EFCore.VisualBasic/SharedTypeExtensions.vb

```diff
@@ -0,0 +1,182 @@
+﻿Imports System.Reflection
+Imports System.Runtime.CompilerServices
+
+<DebuggerStepThrough>
+Module SharedTypeExtensions
+
+    <Extension()>
+    Public Function UnwrapNullableType(type As Type) As Type
+        Return If(Nullable.GetUnderlyingType(type), type)
+    End Function
+
+
```

Comment:
2 blank lines

Curiosity:
This seems to me to be translated here: [http://converter.telerik.com/](http://converter.telerik.com/).

Which means they did a great job 🍻

Reply:
You're absolutely right 🥇 This tool helped me to port the codebase. A few hours weren't wasted doing "translation" :-)